libs/langchain/pyproject.toml

```diff
@@ -151,6 +151,8 @@ pandas = "^2.0.0"
 pytest-mock  = "^3.10.0"
 pytest-socket = "^0.6.0"
 syrupy = "^4.0.2"
+requests-mock = "^1.11.0"
+unstructured = "^0.10.19"
```

Comment:
we should add unstructured to extended_testing group instead of here. you'll want to 
1.  run 
```bash
poetry add --optional unstructured
```
2. add  "unstructured" to extended_testing extras group
3. run
```bash
poetry lock --no-update
```
4. mark the tests that require unstructured with `pytest.mark.requires("unstructured")`

also happy to take over the PR and do it if you'd like!

Reply:
Thanks @baskaryan .
Done + Merged master from upstream

Reply:
@baskaryan you can also take over this PR. 
Thank you 🙏